src/components/Sidebar.js

```diff
@@ -0,0 +1,26 @@
+import React from "react"
+
+export default function Sidebar(props) {
+    const noteElements = props.notes.map((note, index) => (
+        <div key={note.id}>
+            <div
+                className={`title ${
+                    note.id === props.currentNote.id ? "selected-note" : ""
+                }`}
+                onClick={() => props.setCurrentNoteId(note.id)}
+            >
+                <h4 className="text-snippet">Note {index + 1}</h4>
+            </div>
+        </div>
+    ))
+
```

Comment:
@emmyobonyo This should be it's own component

```suggestion

function NoteTile({note, index, currentNoteId, setCurrentNoteId}) {
return (<div key={note.id}>
            <div
                className={`title ${
                    note.id === currentNoteId ? "selected-note" : ""
                }`}
                onClick={() => setCurrentNoteId(note.id)}
            >
                <h4 className="text-snippet">Note {index + 1}</h4>
            </div>
        </div>);
}

export default function Sidebar({ notes, currentNote, newNote, setCurrentNoteId }) {

```

Reply:
I believe it is. It's the `sidebar` component.